ros_cross_compile/dependencies.py

```diff
@@ -95,3 +97,22 @@ def assert_install_rosdep_script_exists(
             'Rosdep installation script has never been created, you need to run this without '
             'skipping rosdep collection at least once.')
     return True
+
+
+class DependenciesStage(PipelineStage):
+    """Wrapper class for running the dependencies stage of the pipeline."""
+
+    def __init__(self):
+        self.name = gather_rosdeps.__name__
```

Comment:
how about `super(DependenciesStage, self).__init__('gather_rosdeps')` - that way we can introduce other setup logic in the future if we want to without depending on knowing how `self.name` is stored/used

Reply:
let's avoid using private attributes

Reply:
name -> _name